tests/clients/test_dapr_grpc_client.py

```diff
@@ -11,7 +11,7 @@
 
 from unittest.mock import patch
 
-from dapr.clients.grpc.client import DaprClient
+from dapr.clients.grpc.client import DaprGrpcClient as DaprClient
```

Comment:
This is odd because it was named correctly before.

Reply:
This is what I mean, you can just name it DaprGrpcClient when using it. So the class can keep the original name.

Reply:
Yeah, but I'd rather not have two classes which are both called DaprClient in the overall project, even if I can alias one of them away. Prefer to just call the DaprGrpcClient the same everywhere and use the generic name DaprClient only for the public facing class that does both Grpc and HTTP. I've updated this to not use an alias, rather just use the name DaprGrpcClient everywhere, so there is no ambiguity anywhere.